### ISSUE TITLE
Preview crashes when cross-referencing an anchor with a :: in the name

### ISSUE THREAD

**Issue by atroccoli** [asciidoctor/atom-asciidoc-preview#51]
To reproduce the problem, type the following text:

```
[[hello::world]]
<<hello::world, Hello World!>>
```

The error reported is
```
Error: Syntax error, unrecognized expression: #hello::world
  at Function.Sizzle.error (c:\Users\atroccoli\AppData\Local\atom\app-0.175.0\resources\app\node_modules\space-pen\vendor\jquery.js:1715:8)
  at tokenize (c:\Users\atroccoli\AppData\Local\atom\app-0.175.0\resources\app\node_modules\space-pen\vendor\jquery.js:2325:11)
  at select (c:\Users\atroccoli\AppData\Local\atom\app-0.175.0\resources\app\node_modules\space-pen\vendor\jquery.js:2712:11)
  at Function.Sizzle (c:\Users\atroccoli\AppData\Local\atom\app-0.175.0\resources\app\node_modules\space-pen\vendor\jquery.js:1154:9)
  at [object Object].jQuery.fn.extend.find (c:\Users\atroccoli\AppData\Local\atom\app-0.175.0\resources\app\node_modules\space-pen\vendor\jquery.js:5163:11)
  at [object Object].jQuery.fn.jQuery.init (c:\Users\atroccoli\AppData\Local\atom\app-0.175.0\resources\app\node_modules\space-pen\vendor\jquery.js:168:38)
  at jQuery (c:\Users\atroccoli\AppData\Local\atom\app-0.175.0\resources\app\node_modules\space-pen\vendor\jquery.js:63:10)
  at AsciiDocPreviewView.module.exports.AsciiDocPreviewView.enableAnchorScroll (c:\Users\atroccoli\.atom\packages\asciidoc-preview\lib\asciidoc-preview-view.coffee:145:21)
  at c:\Users\atroccoli\.atom\packages\asciidoc-preview\lib\asciidoc-preview-view.coffee:128:8
  at Task.callback (c:\Users\atroccoli\.atom\packages\asciidoc-preview\lib\renderer.coffee:30:5)
  at c:\Users\atroccoli\AppData\Local\atom\app-0.175.0\resources\app\src\task.js:57:72
  at Task.module.exports.Emitter.emit (c:\Users\atroccoli\AppData\Local\atom\app-0.175.0\resources\app\node_modules\emissary\lib\emitter.js:118:15)
  at ChildProcess.<anonymous> (c:\Users\atroccoli\AppData\Local\atom\app-0.175.0\resources\app\src\task.js:70:31)
  at ChildProcess.emit (events.js:110:17)
  at handleMessage (child_process.js:323:10)
  at Pipe.channel.onread (child_process.js:351:11)
```

**Comment by anthonny** [comment]
Hi @atroccoli !
Thanks for filing the issue :smiley: 

Maybe i'm wrong, but i think that bug comes from jQuery :
![capture d ecran 2015-01-23 a 00 24 58](https://cloud.githubusercontent.com/assets/2006548/5866911/4efa6b22-a296-11e4-932e-aaa5ee1b6c99.png)

and it's happen when jQuery parse : 
![capture d ecran 2015-01-23 a 00 20 55](https://cloud.githubusercontent.com/assets/2006548/5866921/66e7f5b0-a296-11e4-873a-6b4f0262ff90.png)

IMO, it's a really exotic ID but it's conform to the W3C recommendation http://www.w3.org/TR/html5/dom.html#the-id-attribute

@mojavelinux wdyt ?

**Comment by mojavelinux** [comment]
Does indeed look like a jQuery bug because that is a valid id and the link jump works in a regular HTML document.

Perhaps the problem is how we are using jQuery? Does the id need to be quoted or escaped?

**Comment by Mogztter** [comment]
Resolved in #133
